x-pack/plugins/ml/server/saved_objects/service.ts

```diff
@@ -744,6 +744,8 @@ export function mlSavedObjectServiceFactory(
   }
 
   return {
+    getIds: _getIds,
+    getModelIds: _getModelIds,
```

Comment:
Rather than exposing these functions, it might be better to add functions for explicitly getting AD, DFA and trained model ids.

This is more in keeping with the style of the other exposed functions in this file.
e.g.

```
  async function getAnomalyDetectionJobIds() {
    return _getIds('anomaly-detector', 'job_id');
  }
  async function getDataFrameAnalyticsJobIds() {
    return _getIds('data-frame-analytics', 'job_id');
  }
  async function getTrainedModelsIds() {
    return _getModelIds('model_id');
  }
```